HL_error_covs/errorCovs.py

```diff
@@ -123,8 +125,11 @@ def calc_cov_stats(self, grid_lat, grid_lon, bins, fdbk_var_array, cov_stats, gr
                       lmask1 = np.logical_and(lmask, obs_dist <= bins[nbin-1] + grid_dist[0])
       
                       # Which observations are within the grid square being considered
-                      lmask2 = applyMask.mask_obs_in_grid(lmask, ilon, ilat, grid_lon, 
-                                                          grid_lat, fdbk_var_array)
+                      lmask2 = applyMask.create_mask(lmask, [fdbk_var_array.lons, fdbk_var_array.lons, \
+                                     fdbk_var_array.lats, fdbk_var_array.lats], [grid_lon[ilon]-dlon/2., \
+                                     grid_lon[ilon]+dlon/2., grid_lat[ilat]-dlat/2., grid_lat[ilat]+dlat/2.],
```

Comment:
Does this deal ok with the wrap around points, e.g. when grid_lon[ilon]+dlon/2. > 180 (or 360 depending on how you define it)?